Gateway with gradle doesn't work with Gitlab CI

Please see

https://github.com/jhipster/generator-jhipster/issues/11854

recently closed with 6.9.1

##### **Overview of the issue**

On 6.9.1, the issue is not completely solved. The "package" phase now succeeds, but then the "docker-push" phase fails with an error that looks suspiciously similar to the previous one.

```
./gradlew bootJar -Pprod -x check --no-daemon
 To honour the JVM settings for this build a new JVM will be forked. Please consider using the daemon: https://docs.gradle.org/6.4.1/userguide/gradle_daemon.html.
 Daemon will be stopped at the end of the build stopping after processing
 > Task :bootBuildInfo
 > Task :nodeSetup SKIPPED
 > Task :npmSetup SKIPPED
 > Task :npmInstall UP-TO-DATE
 > Task :webpack FAILED
 FAILURE: Build failed with an exception.
 * What went wrong:
 Execution failed for task ':webpack'.
 > A problem occurred starting process 'command 'npm''
```


-   [X] Checking this box is mandatory (this is just to show you read everything)

<!-- Love JHipster? Please consider supporting our collective:
👉  https://opencollective.com/generator-jhipster/donate -->

--- a/generators/ci-cd/templates/.gitlab-ci.yml.ejs
+++ b/generators/ci-cd/templates/.gitlab-ci.yml.ejs
@@ -153,7 +153,7 @@ gradle-package:
 #    dependencies:
 #        - gradle-package
 #    script:
-#        - ./gradlew jib -Pprod -Djib.to.image=$IMAGE_TAG -Djib.to.auth.username="gitlab-ci-token"  -Djib.to.auth.password=$CI_BUILD_TOKEN
+#        - ./gradlew jib -Pprod -PnodeInstall -Djib.to.image=$IMAGE_TAG -Djib.to.auth.username="gitlab-ci-token"  -Djib.to.auth.password=$CI_BUILD_TOKEN
 
 <%_ if (cicdIntegrations.includes('heroku')) { _%>
 deploy-to-production:

